tools/cellprofiler_v4/cellprofiler.xml

```diff
@@ -0,0 +1,245 @@
+<tool id="cp_cellprofiler4" name="CellProfiler" version="@TOOL_VERSION@+galaxy@VERSION_SUFFIX@">
```

Comment:
Is there a reason here to create a completely new wrapper rather than updating the existing tool?

Reply:
cellprofiler 3 tools has other modules only works with version3. Those module in cp3 suite will not work with cp4.

Reply:
I don't think that is a reason to create a new wrapper. It looks like the function of the tool is the same as the old one, even if the versions are not compatible with each other.

Actually this will create more confusion for the user, because they will search for the tool in Galaxy and see two options, and it won't be immediately clear which to use.